scraper/src/strategies/default_strategy.py

```diff
@@ -187,6 +189,21 @@ def get_records_from_dom(self, current_page_url=None):
                 record['no_variables'] = record['url'] == record[
                     'url_without_variables']
 
+            # Define our own ObjectID to enable proper analytics
+            hierarchy_to_hash = {}
```

Comment:
Prefer a for-comprehension instead of a full-blown loop:
```python
hierarchy_to_hash = {lvl: x for lvl, x in hierarchy.items() if x is not None}
```